### PR TITLE
workout type golfing added

### PR DESCRIPTION
**Pull request by bbbutch**
added a new worktype "golfing". if i missed adding something anywhere, please comment!

### CODE REVIEW

File: translations/de.json [hunk 1]
```diff
@@ -108,6 +108,7 @@
   "show/hide": "Anzeigen/Verbergen",
   "skiing": "Skifahren",
   "snowboarding": "Snowboarden",
+  "golfing": "Golfen",
   "swimming": "schwimmen",
```
Comment (jovandeginste) [inline translations/de.json]: I assume the correct German for is capitalized "Schwimmen"; would you mind adding that extra fix to your PR?

File: pkg/database/workout_type.go [hunk 2]
```diff
@@ -16,14 +16,15 @@ const (
...
 	WorkoutTypeKayaking     WorkoutType = "kayaking"
+	WorkoutTypeGolfing      WorkoutType = "golfing"
 )
 
 func WorkoutTypes() []WorkoutType {
-	return []WorkoutType{WorkoutTypeRunning, WorkoutTypeCycling, WorkoutTypeWalking, WorkoutTypeSkiing, WorkoutTypeSnowboarding, WorkoutTypeSwimming, WorkoutTypeKayaking}
+	return []WorkoutType{WorkoutTypeRunning, WorkoutTypeCycling, WorkoutTypeWalking, WorkoutTypeSkiing, WorkoutTypeSnowboarding, WorkoutTypeSwimming, WorkoutTypeKayaking, WorkoutTypeGolfing}
 }
 
 func DistanceWorkoutTypes() []WorkoutType {
```
Comment (jovandeginste) [inline pkg/database/workout_type.go:26]: Is this actually a distance-based workout? Do you track distance while golfing? Real, honest question, I have no idea!

**Review by jovandeginste — CHANGES_REQUESTED** [review]
two RFC's, see inside

**Comment by jovandeginste** [comment]
You should also add the workout name (`golfing`) to `views/partials/messages.html`, which then makes sure it is added to the translation files (`make generate-messages generate-translations`).

Also run the tailwind update (`make build-tw`), which adds the icons to `output.css`

File: pkg/database/workout_type.go (hunk 2 above)
Comment (bbbutch) [reply pkg/database/workout_type.go:26]: yes, basically it's the distance that you walk on the course between each ball hit.

File: translations/de.json (hunk 1 above)
Comment (bbbutch) [reply translations/de.json]: fixed it

**Review by jovandeginste — APPROVED** [review]

**Comment by jovandeginste** [comment]
Thanks a lot!

**Comment by bbbutch** [comment]
thank you for your help! (and patience with my errors ;) )

**Pull request merged**